cli/src/template_builder.rs

```diff
@@ -165,6 +165,12 @@ pub trait IntoTemplateProperty<'a> {
 
     /// Transforms into a property that will evaluate to `self == other`.
     fn try_into_eq(self, other: Self) -> Option<Box<dyn TemplateProperty<Output = bool> + 'a>>;
+
+    /// Transforms into a property that will evaluate to `self > other`.
+    fn try_into_gt(self, other: Self) -> Option<Box<dyn TemplateProperty<Output = bool> + 'a>>;
+
+    /// Transforms into a property that will evaluate to `self < other`.
+    fn try_into_lt(self, other: Self) -> Option<Box<dyn TemplateProperty<Output = bool> + 'a>>;
```

Comment:
We can add `try_into_cmp() -> .. Output = Ordering` instead? It's unlikely that one of `gt` or `lt` is supported.